src/extension/VSProject.cs

```diff
@@ -371,15 +373,17 @@ private void LoadMSBuildProject()
 
                 if (name == null)
                 {
-                    commonOutputPath = outputPath;
+                    if (outputPathElement != null)

```

Comment:
Could this check be `if (outputPath != null)` instead? I think that's a little less confusing - as `outputPath` is the variable actually used in the assignment. Also - am I right this is a separate bug to the `$(Configuration)` issue? Could you also explain what's going on here - and again add a test for this case? There's a lot going on in this class - and I'm finding it hard to keep tabs on! 😄

Reply:
It can be done like you said - it solves my use case.
The only reason I wrote it like this is protection in case when there is an section with single platform condition with defined path, like this:
  <PropertyGroup Condition=" '$(Platform)' == 'x86' ">
    <OutputPath>\breakes\next\secions\paths\</OutputPath>
  </PropertyGroup>
That section is treated as global configuration (name == 0), but it shouldn't (in my opinion) override commonOutputPath.

You're right, I've started only with $(Configuration) problem, but that didn't solve all my problems, so I've looked deeper ;-)